VISUALIZERS/PLOTLY/BIG_NUMBER/BIG_NUMBER.py

```diff
@@ -56,7 +56,7 @@ def BIG_NUMBER(
         case OrderedPair():
             big_num = default.y[-1]
         case Scalar():
-            big_num = default.c
+            big_num = default.c[-1]
```

Comment:
A scalar should just be a float so this shouldn't be necessary (?) I think there might be something wrong with the app

Reply:
It wasnt the app; when I passed a Scalar type to big number, if the scalar type has length >1, then the line `value=int(float(big_num))` fails (only length-1 arrays can be converted to Python scalars), even though that clearly it should be allowed to display anything from a Scalar data container

Reply:
What was generating a scalar of array-type? This definitely shouldn't happen, it should always be a float

Reply:
It was the constant node, you can avoid the problem by just remembering to set the length to 1, but I made the change to 'user-proof' it and avoid all the potential newcomers that might not have the programming insight to realize you needed to specify length=1